code/modules/power/supermatter/supermatter.dm

```diff
@@ -19,6 +19,7 @@
 #define CO2_HEAT_PENALTY 0.1
 #define NITROGEN_HEAT_PENALTY -1.5
 #define BZ_HEAT_PENALTY 5
+#define FREON_HEAT_PENALTY -10 //very good heat absorbtion and less plasma and o2 generation
```

Comment:
yes, the idea is that the gas must be this strong so that can theoretically stop any and all delamination caused by heating. I've tested it and it slowed down enough a very fast plasma setup delamination so that it could be recovered by anyone